doc/abrt-oops.conf.txt

```diff
@@ -14,13 +14,17 @@ DESCRIPTION
 The configuration file consists of items in the format "Option = Value".
 The following items are recognized:
 
-DropNotReportableOopses = 'yes' / 'no'
-   If you want to see only reportable oopses, set to "yes".
-   Default is 'no': do not drop them.
-
-OnlyFatalMCE = 'yes' / 'no'
-   If you want to see only fatal MCEs, set to "yes".
-   Defaults is 'yes': detect only fatal ones.
+*DropNotReportableOopses = 'yes/no'*::
+   If enabled, only reportable oopses with useful backtraces will be preserved.
+   +
+   Default is 'no', do not drop them.
+
+*OnlyFatalMCE = 'yes/no'*::
+   Many Machine Check Exceptions are corrigible and thus uninteresting to
```

Comment:
Call me stupid but I had to google what `corrigible` means. Why not keep the `correctable`? It is basically the same thing.

Reply:
Indeed, it’s just playing the synonym game.